lib/Payplug/Core/APIRoutes.php

```diff
@@ -62,6 +68,15 @@ public static function setApiBaseUrl($apiBaseUrl)
         self::$API_BASE_URL = $apiBaseUrl;
     }
 
+    /**
+     * @description set $MERCHANT_PLUGINS_DATA_COLLECTOR_RESOURCE from plugin
+     * @param $microServiceBaseUrl
+     */
+    public static function setMerchantPluginsDataCollectiorService($microServiceBaseUrl)
```

Comment:
There is a little typo at Collect.i.or .
And this function are not executed.

Reply:
good catch! i'll fix that